blogs/2016-07-19-mapswipe-app.md

```diff
@@ -0,0 +1,45 @@
+---
+title: MapSwipe app lets public help deliver medical care to most vulnerable
+publishedDate: 2016-07-19
+author: Médecins Sans Frontières (MSF)
+description: Médecins Sans Frontières (MSF) unveils a first-of-its-kind mobile phone app in which people across the globe can locate communities in remote parts of the world affected by natural disasters, disease outbreaks or conflict.
+coverImage: /img/blogImages/
```

Comment:
@danbjoseph , the build is failing because the url doesn't have an image.
You'll need to add appropriate image here as well.

Reply:
ah, i see. #199 should address this.